orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -232,6 +237,12 @@ def _build_graph(self):
 
     def _get_session_config(self):
         """Creates and returns the session configuration to be used for the ExecutionAgent"""
+
+        if torch.are_deterministic_algorithms_enabled():
+            if self._debug_options.logging.log_level <= _logger.LogLevel.INFO:
+                warnings.warn("ORTModule's determinism will be enabled because PyTorch's determinism is enabled.",
```

Comment:
Like it 👍

Reply:
Actually, did you mean to write >= INFO? https://docs.python.org/3/library/logging.html#levels